br/pkg/restore/data.go

```diff
@@ -346,25 +421,47 @@ func (recovery *Recovery) makeRecoveryPlan() {
 		rid    uint64
 	}
 
-	// Group region peer info by region id.
+	// Group region peer info by region id. find the max allcateId
 	// region [id] [peer[0-n]]
 	var regions = make(map[uint64][]Regions, 0)
 	for _, v := range recovery.regionMetas {
```

Comment:
```suggestion
	for _, v := range recovery.recoveryMetas {
```

Reply:
seems recovery meta would be more approviate

Reply:
changed from recoverymeta to regionmeta, since they are just a regionmeta for recovery. it does not need to recovery meta of region.

Reply:
as your another comments already stated, storeMetas may be better.